### PR TITLE
[DEM-926] An example for scan1D with UHFLI

### PR DESCRIPTION
**Pull request by qSaevar**
provides ./docs/notebooks/unfinished/example_measurement_uhfli.py

### CODE REVIEW

File: src/tests/measurements/acquisition/test_uhfli_scope_reader.py [hunk 1]
```diff
@@ -311,3 +311,21 @@ def test_input_signal_is_correctly_set(self):
...
+        scope_mock.acquire_single_sample(2, 'x')
+
+        scope_mock.adapter.instrument.demod2_x.assert_called_once_with()
+
+    def test_acquire_single_sample_partial(self):
+        mock_address = 'dev2350'
+        scope_mock, _ = TestUhfliScopeReader.__patch_scope_reader(mock_address)
+        self.assertEqual(mock_address, scope_mock.adapter.address)
+
+        paramter = scope_mock.acquire_single_sample(2, 'x', partial=True)
```
Comment (CumulonimbusCalvus) [inline src/tests/measurements/acquisition/test_uhfli_scope_reader.py]: parmeter --> parameter?

File: src/tests/measurements/acquisition/test_uhfli_stimulus.py [hunk 2]
```diff
@@ -52,6 +52,16 @@ def test_set_oscillator_frequency(self):
         expected_calls = [call('oscillator5_freq'), call()(42.0)]
         yamock.assert_has_calls(expected_calls)
 
+    def test_set_oscillator_frequency_partial(self):
+        yamock = MagicMock()
```
Comment (CumulonimbusCalvus) [inline src/tests/measurements/acquisition/test_uhfli_stimulus.py]: What is a yamock?

Comment (qSaevar) [reply src/tests/measurements/acquisition/test_uhfli_stimulus.py]: yet another mock

File: src/tests/measurements/acquisition/test_uhfli_scope_reader.py (hunk 1 above)
Comment (qSaevar) [reply src/tests/measurements/acquisition/test_uhfli_scope_reader.py]: ✔️

File: src/tests/measurements/acquisition/test_uhfli_stimulus.py (hunk 2 above)
Comment (qSaevar) [reply src/tests/measurements/acquisition/test_uhfli_stimulus.py]: I changed it.

**Pull request merged**